[2024-04-26T04:55:12.608+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-26T04:55:12.627+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Crypto_ETL.fetch_and_save_data manual__2024-04-26T04:19:49.652646+00:00 [queued]>
[2024-04-26T04:55:12.635+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Crypto_ETL.fetch_and_save_data manual__2024-04-26T04:19:49.652646+00:00 [queued]>
[2024-04-26T04:55:12.637+0000] {taskinstance.py:2303} INFO - Starting attempt 4 of 4
[2024-04-26T04:55:12.652+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): fetch_and_save_data> on 2024-04-26 04:19:49.652646+00:00
[2024-04-26T04:55:12.660+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=4261) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-26T04:55:12.662+0000] {standard_task_runner.py:63} INFO - Started process 4263 to run task
[2024-04-26T04:55:12.662+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Crypto_ETL', 'fetch_and_save_data', 'manual__2024-04-26T04:19:49.652646+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/processing_ETL.py', '--cfg-path', '/tmp/tmpbbnl9y6u']
[2024-04-26T04:55:12.663+0000] {standard_task_runner.py:91} INFO - Job 24: Subtask fetch_and_save_data
[2024-04-26T04:55:12.717+0000] {task_command.py:426} INFO - Running <TaskInstance: Crypto_ETL.fetch_and_save_data manual__2024-04-26T04:19:49.652646+00:00 [running]> on host e30f8265f0c9
[2024-04-26T04:55:12.858+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Crypto_ETL' AIRFLOW_CTX_TASK_ID='fetch_and_save_data' AIRFLOW_CTX_EXECUTION_DATE='2024-04-26T04:19:49.652646+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-26T04:19:49.652646+00:00'
[2024-04-26T04:55:12.860+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-26T04:55:12.874+0000] {logging_mixin.py:188} INFO - Data saved to binance_dataset.csv
[2024-04-26T04:55:12.875+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-04-26T04:55:12.876+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-26T04:55:12.889+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=Crypto_ETL, task_id=fetch_and_save_data, execution_date=20240426T041949, start_date=20240426T045512, end_date=20240426T045512
[2024-04-26T04:55:12.917+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-26T04:55:12.934+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-26T04:55:12.936+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
